targets/targets.json

```diff
@@ -1700,6 +1700,15 @@
         "release_versions": ["2", "5"],
         "device_name": "STM32L072CZ"
     },
+    "CMWX1ZZABZ_078": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M0+",
+        "extra_labels_add": ["STM32L0", "STM32L0x2xB", "STM32L082KB", "STM32L082xx"],
+        "detect_code": [],
+        "device_has_add": ["ANALOGOUT", "LOWPOWERTIMER", "SERIAL_FC", "SERIAL_ASYNCH", "TRNG", "FLASH"],
```

Comment:
Add macro MBEDTLS_CONFIG_HW_SUPPORT ?

Reply:
added

Reply:
Actually not as AES driver does not support STM32L0x2 yet.